### PR TITLE
Add sestegra as an fr approver

### PR DESCRIPTION
**Pull request by huats**
### Describe your changes
Given his active collaboration in French localization team, this PR recognizes the effort made by @sestegra and promotes him to the approver role.

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -166,6 +166,11 @@ collaborators:
   - username: Krast76
     permission: push
 
+  - username: sestegra
+    permission: push
+
+  # l10n ur approvers
+
   # l10n ur approvers
```
[the remark this answers is not in the document]
Comment (huats) [reply .github/settings.yml]: Pfff what a mistake...  You should never push a PR before catching a plane in a hurry :) Thanks for catching

**Comment by seokho-son** [comment]
LGTM!

Hi @sestegra welcome to the French approver team!
please check and confirm following policies :)
[Policies for localization approvers (to be confirmed by candidates) #723](https://github.com/cncf/glossary/discussions/723)

**Comment by sestegra** [comment]
Hi @seokho-son, @huats 
Thank you!
I understand and agree to the policies.

**Review by seokho-son — APPROVED** [review]
lgtm! :)

**Review by jihoon-seo — APPROVED** [review]
LGTM! Welcome!

**Pull request merged**